### PR TITLE
chore: Prisma 5 upgrade NestJs `beforeExit` upgrade note

### LINKED ISSUE #5066: Note for NestJs `beforeExit` hook removal on upgrading to Prisma 5

### PR DESCRIPTION
**Pull request by ruheni**
Closes #5066

### CODE REVIEW

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx [hunk 1]
```diff
@@ -661,6 +661,23 @@ process.on('SIGTERM', exitHandler)
 process.on('SIGUSR2', exitHandler)
 ```
 
+If you're using the `beforeExit` hook in NestJs, you can upgrade to Prisma 5 by removing the `enableShutdownHooks` in your application:
```
Comment (aqrln) [inline content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx]:
```suggestion
If you're using the `beforeExit` hook in NestJS, you can upgrade to Prisma 5 by removing the custom `enableShutdownHooks` method in your application:
```

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx [hunk 2]
```diff
@@ -661,6 +661,23 @@ process.on('SIGTERM', exitHandler)
...
+    await this.$connect()
+  }
+
+-  async enableShutdownHooks(app: INestApplication) {
+-    this.$on('beforeExit', async () => {
+-      await app.close()
+-    })
+-  }
+}
+```
```
Comment (aqrln) [inline content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx:679]:
````suggestion
```

Instead, use the built-in `enableShutdownHooks` method in NestJS if you need to handle lifecycle events:

```diff file="main.ts"
- prismaService.enableShutdownHooks(app)
+ app.enableShutdownHooks()
```
````

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx [hunk 3]
```diff
@@ -661,6 +661,30 @@ process.on('SIGTERM', exitHandler)
 process.on('SIGUSR2', exitHandler)
 ```
 
+If you're using the `beforeExit` hook in NestJS, you can upgrade to Prisma 5 by removing the custom `enableShutdownHooks` method in your application:
```
Comment (jharrell) [inline content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx]:
```suggestion
If you're using the `beforeExit` hook in NestJS, you can upgrade to Prisma 5 by removing the custom `enableShutdownHooks` method in your service:
```

Comment (jharrell) [reply content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx]: Not sure on capitalization... as per usual :)

**Review by jharrell — APPROVED** [review]

**Pull request merged**